samples/ChatSample/Hubs/Chat.cs

```diff
@@ -20,7 +20,7 @@ public Chat(IUserTracker<Chat> userTracker)
         {
             if (!Context.User.Identity.IsAuthenticated)
             {
-                Context.Connection.Dispose();
```

Comment:
A general comment: How is the user supposed to know that they should do that? What would happen if they threw an exception - would the transport be disposed?